bodhi/__init__.py

```diff
@@ -0,0 +1 @@
+__import__('pkg_resources').declare_namespace(__name__)
```

Comment:
I just dropped this and tested it both in a development environment and a built RPM, and things still work.